### PR TITLE
fix for isEnabled renaming due to parameter clash

### PR DESCRIPTION
**Pull request by vpodlesnyak**

### CODE REVIEW

File: stencils/KalugaButton+SwiftUI.stencil [hunk 1]
```diff
@@ -26,7 +26,7 @@ extension KalugaButton {
                 buttonView
             }
         )
-        .buttonStyle(self.style, isEnabled: self.isEnabled)
+        .buttonStyle(self.style, isEnabled: self.isEnabled_)
```
Comment (cikpis) [inline stencils/KalugaButton+SwiftUI.stencil:29]: This again? I thought it was fixed using the last Kaluga + KalugaSwiftUI

Comment (vpodlesnyak) [reply stencils/KalugaButton+SwiftUI.stencil:29]: apparently not

Comment (vpodlesnyak) [reply stencils/KalugaButton+SwiftUI.stencil:29]: @cikpis , @Daeda88 any idea on why the issue is back?

**Review by Daeda88 — CHANGES_REQUESTED** [review]
Rejecting: this is not a fix as it only pops up when exporting the bluetooth module to an iOS project. Can be fixed through gradle on the project or by renaming isEnabled on kaluga.

**Pull request closed without merging**